### PR TITLE
Installing on Windows with winget

### PR DESCRIPTION
**Pull request by TrangOul**

### CODE REVIEW

File: doc/windows.md [hunk 1]
```diff
@@ -15,7 +15,14 @@ Download the [latest release]:
 
 and extract it.
 
-Alternatively, you could install it from packages manager, like [Chocolatey]:
+Alternatively, you could install it from packages manager, like [Winget]:
+
+```bash
+winget install scrcpy
+winget install adb    # if you don't have it yet
```
Comment (rom1v) [inline doc/windows.md:22]: Is `adb` available via Winget? I did not found it in https://github.com/microsoft/winget-pkgs

Comment (yume-chan) [reply doc/windows.md:22]: `winget install scrcpy` should be enough, it uses the official Windows release which already includes adb.exe.

adb does exist in winget packages, but can only be installed by its ID:

```sh
winget install Google.PlatformTools
```

Comment (rom1v) [reply doc/windows.md:22]: OK, so I will just remove the line about adb.

Do you know if it is necessary to install adb separately with scoop and chocolatey package managers, or if the scrcpy package is sufficient?

Comment (yume-chan) [reply doc/windows.md:22]: I don't know, I don't use other package managers.

**Comment by rom1v** [comment]
Thank you, merged: 958f22490b3ab86677bd8b6126ddaa4de4082798

(I removed the line about `adb`)

**Comment by TrangOul** [comment]
My bad about `adb`, thanks for correcting that!

**Pull request closed without merging**